sass/_themes.scss

```diff
@@ -106,6 +106,8 @@ $COLOR_CORAL_RED  : #ff444f;
 $COLOR_SKY_BLUE   : #2196f3;
 $COLOR_WHITE      : #ffffff;
 $COLOR_BLUE       : #1c5ae3;
+$COLOR_CHART_RISE_UP: rgba(249, 84, 84, 0.32);
+$COLOR_CHART_FALL_DOWN: rgba(76, 175, 80, 0.32);
```

Comment:
@salar-fs  as designer mentioned here https://zpl.io/awoMowd (the comment #2) closed color should be 0.32 opacity of original colors, so we don't need these two variables, we can follow your pattern here `rgba($color-red, 0.32)`